src/main/scala/mesosphere/marathon/tasks/ResourceUtil.scala

```diff
@@ -151,15 +151,26 @@ object ResourceUtil {
     }
   }
 
-  def displayResource(resource: Resource): String = resource.getType match {
-    case Value.Type.SCALAR => s"${resource.getName} ${resource.getScalar.getValue}"
-    case Value.Type.RANGES =>
-      s"${resource.getName} ${
-        resource.getRanges.getRangeList.asScala.map {
-          range => s"${range.getBegin}->${range.getEnd}"
-        }.mkString(",")
-      }"
-    case other: Value.Type => resource.toString
+  def displayResource(resource: Resource): String = {
+    // TODO AW: place it somewhere appropriate
+    val MaxRangesToDisplay = 10
```

Comment:
@aquamatthias I changed the output of large port lists to look like this:
`ports 1->2,3->4,5->6,7->8,9->10,11->12,13->14,15->16,17->18,19->20 ... (90 more)`
Are you ok with this? If so, I am not quiet sure where to put the upper limit for ranges to display. As you can see, I've placed a constant number for now, right into the display function.